test/Microsoft.Framework.DependencyInjection.Tests/ActivatorUtilitiesTests.cs

```diff
@@ -79,12 +79,24 @@ public void TypeActivatorWorksWithStaticCtor(CreateInstanceFunc createFunc)
             Assert.NotNull(anotherClass);
         }
 
-        [Theory]
-        [MemberData(nameof(CreateInstanceFuncs))]
-        public void TypeActivatorWorksWithCtorWithOptionalArgs(CreateInstanceFunc createFunc)
```

Comment:
I don't think you should have changed/split the test. It wouldn't be difficult to make the old test pass.

When you are fixing CreateFactory to match CreateInstance's behavior in one scenario, I don't think you should be breaking the consistency of another behavior unless there is a really good reason for doing so.